src/utils/stop-scrolling.ts

```diff
@@ -0,0 +1,7 @@
+export function stopScrolling() {
+    document.body.style.overflow = 'hidden';
```

Comment:
which browsers / platforms was this tested on? (I assume you tested this manually)

Reply:
Chrome, Safari, Firefox, Edge, and IE11